<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateMatching</class>
 <widget class="QMainWindow" name="TemplateMatching">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TemplateMatching</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="openFileButton2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开图片2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFileButton1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开图片1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>偏差值：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_offset">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_filename1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>511</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_filename2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>511</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_calDif">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>计算偏差</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_refPos">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_refSize">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>190</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>样本框位置：（空格分隔，如：10 20。下同）</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>样本框尺寸：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_help">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_selectRef">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>样本框选取</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_showEffect">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>显示修正效果</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_canny1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>边缘检测1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_canny2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>边缘检测2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Overallscan">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>是否全图扫描</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
